doc/source/getting_started/installing.rst

```diff
@@ -0,0 +1,458 @@
+.. _installing_odl:
+
+##############
+Installing ODL
+##############
+
+************
+Introduction
+************
+
+Installing ODL is intended to be straightforward, and this guide is meant for new users.
+For a minimal installation you should perform the following steps:
+
+1. Install a Python interpreter
+2. Install ODL and its dependencies
+3. (optional) Run the tests
+
```

Comment:
I actually don't want to do that. Most Linux people have a Python interpreter, but I don't want them to miss the Anaconda recommendation. And they should know that they can skip things that they have installed themselves manually at some point.